ibis/backends/bigquery/tests/system/snapshots/test_client/test_cross_project_query/out.sql

```diff
@@ -1,6 +1,6 @@
 SELECT t0.`title`, t0.`tags`
 FROM (
-  SELECT t1.*
-  FROM `bigquery-public-data.stackoverflow.posts_questions` t1
-  WHERE STRPOS(t1.`tags`, 'ibis') - 1 >= 0
+  SELECT t2.*
+  FROM `bigquery-public-data.stackoverflow.posts_questions` t2
+  WHERE STRPOS(t2.`tags`, 'ibis') - 1 >= 0
```

Comment:
does this indicate that we're generating an intermediary alias and then dropping it along the way?

Reply:
It's possible the +1 on the parent isn't necessary, let me check.

Reply:
Looks like that isn't needed, since `make_alias` leaves the `_alias_counter` at the value of the *next* alias.

Reply:
Which means the number of regenerated files should decrease!